public/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta http-equiv="Expires" content="0" />
+    <title>Welcome</title>
+    <script src="https://officedev.github.io/custom-functions/lib/office.js"></script>
+    <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.6.0/css/fabric.min.css">    
+
+    <script>
+        Office.onReady(function() { /* nothing to initialize */ })
+    </script>
+</head>
+
+<body class="ms-Fabric">
+    <p>
+        You can use this sample to try Custom Functions.
```

Comment:
So custom functions does not need caps - both can be lowercase when not in a title.